app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  validates :content, presence: true, length: { maximum: 150 }
+  validates :post, presence: true
```

Comment:
We don't need to validate anything that is already validated by the `belongs_to` association.

For instance, post and user are already in the below associations.